docs/CeedlingPacket.md

```diff
@@ -985,6 +985,40 @@ that you execute on target hardware).
 
   **Default**: FALSE
 
+  ---
+
+  **Note:**
+
+    The configuration can be combined together with
+    
+    ```
+    :test_runner:
+        :cmdline_args: true
+    ```
+
+    After setting **cmdline_args** to **true**, the debuger will execute each of test
+    case from crashing test file separately. The exclude and include test_case patterns will
+    be applied, to filter execution of test cases.
+
+    The .gcno and .gcda files will be generated and section of the code under test case
+    causing segmetation fault will be omitted from Coverage Report.
+
+    The default debugger (**gdb**)[https://www.sourceware.org/gdb/] can be switch by any other
+    via setting new configuration under tool node in project.yml. At default set to:
+
+    ```
+      :tools:
+        :backtrace_settings:
+          :executable: gdb
+          :arguments: [ "-q --eval-command run --eval-command backtrace --batch --args" ]
```

Comment:
in other cases this is presented as array of argument/value pairs. Its best to put it same here

```suggestion
          :arguments:
            - -q
            - --eval-command run
            - --eval-command backtrace
            - --batch
            - --args
```

Reply:
Hi @Letme ,

I though about writing documentation in the same way as you mentioned, but I didn't wanted to confuse end engineer wit implementation. 

The arguments node it need to be a list, as is giving use possibility list arguments as you mentioned one after second one.

As well the Ceedling implementation looks like this:

1. Load default settings
2. Load settings from project.yml
3. make a diff and apply diff from project.yml in default settings.

To avoid adding not compatible gdb argument to any other debugger executable, we drop the first element, and use passed, defined by user in project.yml.

So mostly when someone configure:

```yaml
:tools:
        :backtrace_settings:
          :executable: gdb
          :arguments: [ "-q --eval-command run --eval-command backtrace --batch --args" ]
```

in project.yml will totally overwritte current configuration.

----

 @mvandervoord  : Is my solution acceptable by you? With overwriting **:arguments** array?

Reply:
Myself, I prefer the consistency of arguments being one-per line. I've been thinking that we should allow an array of options OR a string (not in an array) with all the options on a single line for ALL the tools. But until we do that, I'd rather not do something inconsistent.

Reply:
Hi @mvandervoord ,

The documentation was fixed after latest changes